app.py

```diff
@@ -0,0 +1,62 @@
+from flask import Flask, render_template, request, redirect, url_for, flash
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///database.db'
+app.config['SECRET_KEY'] = 'mysecretkey'
+db = SQLAlchemy(app)
+
+class Todo(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(100), nullable=False)
+    completed = db.Column(db.Boolean, default=False)
+
+    def __repr__(self):
+        return f'<Todo {self.id} - {self.title}>'
+
+@app.route('/')
+def home():
+    todos = Todo.query.all()
+    error = None
+    success = None
```

Comment:
If both error and success will always be None, Then there isn't any need for initializing these variables.